test/paper-dialog-behavior.html

```diff
@@ -333,45 +327,34 @@ <h2 id="header">Dialog</h2>
         });
 
         test('multiple modal dialogs opened, handle focus change', function(done) {
-          ensureDocumentHasFocus();
-
           var dialogs = fixture('multiple');
-          var focusChange = sinon.stub();
-          document.body.addEventListener('focus', focusChange, true);
+          var spy = sinon.spy(dialogs[1]._focusNode, 'focus');
 
           runAfterOpen(dialogs[0], function() {
-            // Wait 10ms to allow focus changes
-            dialogs[1].async(dialogs[1].open, 10);
-            dialogs[1].addEventListener('iron-overlay-opened', function() {
-              // Wait 10ms to allow focus changes
-              setTimeout(function() {
-                // Should not enter in an infinite loop.
-                assert.equal(focusChange.callCount, 2, 'focus change count');
+            runAfterOpen(dialogs[1], function() {
+              // Wait 50ms, should not enter in an infinite loop.
```

Comment:
Oh, hmm, I see. Can you maybe add that as a comment as to why we're maybe fearing this infinite loop?